lib/seahorn/BmcPass.cc

```diff
@@ -362,6 +362,43 @@ class BmcPass : public llvm::ModulePass {
       }
     }
 
+    // -- compute dependence of verifier.assume.not()
```

Comment:
comment and code disagree. Comment talks about `assume` but code about `assert.if`. Yet `assumeFn` variable indicates that this is an `assume` and not an `assert`

Reply:
fixed